packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -298,7 +300,14 @@ const DocumentActionsMenu = ({
                   gap={2}
                   tag="span"
                 >
-                  <span>{action.icon}</span>
+                  <Flex
+                    tag="span"
+                    color={
+                      !action.disabled ? convertActionVariantToIconColor(action.variant) : 'inherit'
```

Comment:
That logic to get the color is the same as in the one in the parent Flex above. So if you set inherit as the color here, regardless of the disabled state, don't you get the same result?

Reply:
no its undefined but secondary where it was breaking, thats why we have `convertActionVariantToIconColor` as a separate function.